libs/designsystem/header/src/header.component.html

```diff
@@ -7,59 +7,72 @@
     <ng-content select="kirby-flag"></ng-content>
   </ng-container>
 
-   <!-- Custom Content in Flag section: -->
+  <!-- Custom Content in Flag section: -->
   <ng-container *ngIf="!!customFlagTemplate">
     <div class="custom-flag">
-        <ng-container *ngTemplateOutlet="customFlagTemplate"></ng-container>
+      <ng-container *ngTemplateOutlet="customFlagTemplate"></ng-container>
     </div>
   </ng-container>
 </div>
 
 <div class="container">
-    <!-- Title and value - title: -->
-  <h1 *ngIf="!!title && !!value" #titleElement class="title kirby-text-medium" [class.clickable]="hasInteractiveTitle !== false && titleClick.observed" [class.has-icon]="!!titleActionIconTemplate" (click)="onTitleClick($event)">
+  <!-- Title and value - title: -->
+  <span
+    *ngIf="!!title && !!value"
```

Comment:
Consider using Angular's control flow instead and forego the old structural directives 😎

Reply:
We are looking into migrating all of `/libs/designsystem` components at once to not have different coding styles throughout, so will postpone until then!